notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>Can you provide a link to a paper, or a blog post that describes the scores, what are relatedness, relevance, standalone?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>I added a link to a Google Paper, and renamed "relatedness" into "groundedness" to match the paper (the paper also refers relevance). "Standalone"is my creation, I did not find it in the literature: it aims to eliminate the third possible flaw, "an answer tailored for the particular setting where the question was generated, but undecipherable by itself, like&nbsp;<code>"What is the name of the function used in this guide?"</code> "</p><p>Btw I'd be interested if you have a better idea of a name for this criterion than "Standalone" 👀</p><p> </p>